src/types/api.js

```diff
@@ -141,8 +141,9 @@
  * TODO - Fill out remaining properties
  *
  * @typedef Group
- * @prop {string} id
- * @prop {string} groupid
+ * @prop {string} id - a.k.a. "pubid", unique per authority.
+ * @prop {string} [groupid] - Fully-qualified ID with authority.
+ *   Not all groups have a `groupid`. LMS groups always have a `groupid`.
```

Comment:
This distinction between `id` and `groupid` is super duper important for these changes, and I worry confusion over the two could trip up future us. You probably are quite familiar with `id` (a.k.a. `pubid`). The list of groups passed to the RPC `changeFocusModeUser` method is an Array of `groupid`s — fully-qualified, unique-across-all-authorities group IDs that look like `group:<md5 hash>@lms.hypothes.is`.

All LMS groups have a `groupid`, but first party groups do not.